SMDS_3/include/CGAL/IO/File_medit.h

```diff
@@ -961,7 +961,7 @@ output_to_medit(std::ostream& os,
  * \cgalNamedParamsBegin
  * \cgalParamNBegin{all_cells}
  *   \cgalParamDescription{If `true`, all the cells in `t3` are written in `os`,
- *        whether they belong to the complex or not.
+ *        however they belong to the complex or not.
```

Comment:
"whether" was correct (and there is another "however" farther down)

Reply:
I changed "whether" to "however" to have both consistent :-)
IMHO we don't really want to say "if" but "no matter what"

Reply:
I agree "however they belong to" sound strange.